x-pack/plugins/siem/public/components/open_timeline/note_previews/note_preview.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiAvatar, EuiFlexGroup, EuiFlexItem, EuiText, EuiSpacer, EuiToolTip } from '@elastic/eui';
+import { FormattedRelative } from '@kbn/i18n/react';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { FormattedDate } from '../../formatted_date';
+import { getEmptyValue } from '../../empty_value';
+import { Markdown } from '../../markdown';
+
+import * as i18n from '../translations';
+
+interface Props {
+  note?: string | null;
+  updated?: number | null;
+  updatedBy?: string | null;
+}
+
+export const Avatar = styled(EuiAvatar)`
+  margin-right: 12px;
+  user-select: none;
+`;
+
+const UpdatedBy = styled.div`
+  font-weight: bold;
+`;
+
+const NotePreviewFlexGroup = styled(EuiFlexGroup)`
+  margin: 16px 0;
+  width: 100%;
+`;
+
+/**
+ * Renders a preview of a note in the All / Open Timelines table
+ */
+export const NotePreview = pure<Props>(({ note, updated, updatedBy }) => (
+  <NotePreviewFlexGroup direction="row" gutterSize="none">
+    <EuiFlexItem grow={false}>
+      <Avatar data-test-subj="avatar" size="m" name={updatedBy != null ? updatedBy : '?'} />
+    </EuiFlexItem>
+
+    <EuiFlexItem grow={true}>
+      <UpdatedBy data-test-subj="updated-by">
+        {updatedBy != null ? updatedBy : getEmptyValue()}
+      </UpdatedBy>
```

Comment:
There is a handy helper created a while back you can use here called, `defaultToEmptyTag`, it works like this:

```ts
<UpdatedBy data-test-subj="updated-by">defaultToEmptyTag(updatedBy)</UpdatedBy>
```

And it will perform not only the check against a `null` but also if for some reason `updatedBy` is an empty string it will add the EmptyString placeholder.